app/src/main/java/com/arygm/quickfix/ui/Map.kt

```diff
@@ -1,35 +1,81 @@
 package com.arygm.quickfix.ui
 
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
 import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
 import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.shape.RoundedCornerShape
 import androidx.compose.material3.ExperimentalMaterial3Api
 import androidx.compose.material3.Scaffold
+import androidx.compose.material3.Surface
 import androidx.compose.material3.Text
 import androidx.compose.material3.TopAppBar
+import androidx.compose.material3.TopAppBarDefaults
 import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.unit.dp
+import com.arygm.quickfix.ui.navigation.BottomNavigationMenu
 import com.arygm.quickfix.ui.navigation.NavigationActions
 import com.arygm.quickfix.ui.navigation.Route
 
 @OptIn(ExperimentalMaterial3Api::class)
 @Composable
-fun MapScreen(navigationActions: NavigationActions, isUser: Boolean = true) {
-  // Use Scaffold for the layout structure
-  Scaffold(
-      topBar = { TopAppBar(title = { Text(text = "MAP Screen") }) },
-      bottomBar = {
-        // Boolean isUser = true for this HomeScreen
-        MeowBottomNavigationMenu(
-            selectedItem = Route.MAP, // Start with the "Home" route
-            onTabSelect = { selectedDestination ->
-              // Use this block to navigate based on the selected tab
-              navigationActions.navigateTo(selectedDestination)
-            },
-            isUser = isUser // Assuming the user is of type User
+fun MapScreen(navigationActions: NavigationActions, isUser: Boolean = true, LoD: Boolean) {
+
+    val color1 = if (LoD) Color(0xFFF16138) else Color(0xFF633040)
+    val backgroundColor = if (LoD) Color.White else Color(0xFF282828)
```

Comment:
color1 and backgroundColor could be more descriptive. For example, primaryColor and screenBackgroundColor would clarify their roles.